lib/view_model/send/send_view_model.dart

```diff
@@ -127,15 +130,18 @@ abstract class SendViewModelBase with Store {
   bool get isReadyForSend => _wallet.syncStatus is SyncedSyncStatus;
 
   @computed
-  ObservableList<Template> get templates => sendTemplateViewModel.templates;
+  ObservableList<Template> get templates => sendTemplateViewModel.templates
+      .where((template) => template.cryptoCurrency == _wallet.currency.title)
+      .toList()
+      .asObservable();
```

Comment:
Not sure that here is needed `ObservableList` as you use `computed` then it will regenerate the result of getter body in any case. Keep `ObservableList` only if you manually edit content of exactly special list. Better to use just `List` here.

Reply:
okay, done